lib/babel.js

```diff
@@ -1,22 +1,30 @@
 'use strict';
-var resolveFrom = require('resolve-from');
+var unsafeResolveFrom = require('resolve-from');
 var createEspowerPlugin = require('babel-plugin-espower/create');
 var requireFromString = require('require-from-string');
+var path = require('path');
 var hasGenerators = parseInt(process.version.slice(1), 10) > 0;
-var path = process.argv[2];
+var testFile = process.argv[2];
 
-var babel;
-
-try {
-	var localBabel = resolveFrom('.', 'babel-core') || resolveFrom('.', 'babel');
-	babel = require(localBabel);
-} catch (err) {
-	babel = require('babel-core');
+function resolveFrom(dir, p) {
+	try {
+		return unsafeResolveFrom(dir, p);
+	} catch (e) {}
 }
 
+var babelPath = resolveFrom('.', 'babel-core') || resolveFrom('.', 'babel') || require.resolve('babel-core');
+var babel = require(babelPath);
+
+var rtp1 = 'babel-runtime/regenerator';
+var rtp2 = 'regenerator/runtime-module';
+var rtp3 = 'babel-core/node_modules/regenerator/runtime-module';
+var babelDir = path.dirname(babelPath);
+var runtimePath = resolveFrom('.', rtp1) || resolveFrom('.', rtp2) || resolveFrom('.', rtp3) || resolveFrom(babelDir, rtp1) || resolveFrom(babelDir, rtp2) || resolveFrom(babelDir, rtp3) || require.resolve(rtp1);
+var bluebirdPath = require.resolve('bluebird');
```

Comment:
@jamestalmage Can you open an issue about it over at core.js? Would be more convenient to use that one, otherwise we'll need to figure out a way to manually alias it to pinkie.